content/en/database_monitoring/setup_oracle/rac.md

```diff
@@ -25,13 +25,36 @@ Database Monitoring provides deep visibility into your Oracle databases by expos
 
 Complete the following steps to enable Database Monitoring with your database:
 
-1. Configure the Agent for each RAC node by following the instructions for [self-hosted Oracle databases][7].
+Configure the Agent for each RAC node by following the instructions for [self-hosted Oracle databases][7].
 
-2. Add the following custom tag to each node:
-    ```yaml
-    tags:
-      - rac_cluster:your_cluster_name
-    ```
+You must configure the Agent for each RAC node, because the Agent collects information from every node separately by querying `V$` views. The Agent doesn't query any `GV$` views to avoid generating interconnect traffic. The collected data from each RAC node is aggregated in the frontend.
```

Comment:
Recommend spelling out the acronym the first time it's used.
```suggestion
You must configure the Agent for each Real Application Cluster (RAC) node, because the Agent collects information from every node separately by querying `V$` views. The Agent doesn't query any `GV$` views to avoid generating interconnect traffic. The collected data from each RAC node is aggregated in the frontend.
```